packages/combo-box/test/toggling-dropdown.test.js

```diff
@@ -127,22 +127,31 @@ describe('toggling dropdown', () => {
       expect(comboBox.opened).to.be.true;
     });
 
-    it('should be hidden with null items', () => {
+    it('should not open overlay when setting items to null', () => {
       comboBox.items = null;
 
       comboBox.open();
 
       expect(comboBox.opened).to.be.true;
-      expect(overlay.hasAttribute('hidden')).to.be.true;
+      expect(overlay.opened).to.be.false;
     });
 
-    it('should be hidden with no items', () => {
+    it('should not open overlay when setting empty items array', () => {
       comboBox.items = [];
 
       comboBox.open();
 
       expect(comboBox.opened).to.be.true;
-      expect(overlay.hasAttribute('hidden')).to.be.true;
+      expect(overlay.opened).to.be.false;
+    });
+
+    it('should not open overlay when setting empty filteredItems array', () => {
```

Comment:
Is it tested somewhere that the overlay finally opens after you provide some items?